src/Client.php

```diff
@@ -579,7 +579,7 @@ public function getProductVariations(array $productIds, array $with, $storePlent
         ];
 
         if ($storePlentyId) {
```

Comment:
This information is from the `rest/webstores` route, and it is the id of a webstore. I want to leave it like that, since it's just the query parameter for that route, that is different.
![image](https://user-images.githubusercontent.com/20555112/92233890-61768200-eeb1-11ea-8808-0126943a6d28.png)